src/ensembl/src/content/app/browser/track-panel/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -0,0 +1,188 @@
+import type { TrackActivityStatus } from 'src/content/app/browser/track-panel/trackPanelConfig';
+import type { RootState } from 'src/store';
+
+import styles from './TrackPanelItem.scss';
+
+type TrackPanelGeneProps = {
+  genomeId: string;
+  geneId: string;
+  ensObjectId: string;
+};
+
+// TODO: figure out proper gene and transcript track naming conventions
+const GENE_TRACK_ID = 'track:gene-feat';
+const getTranscriptTrackId = (num: number) => `track:transcript-feat-${num}`;
+
+const TrackPanelGene = (props: TrackPanelGeneProps) => {
+  const { genomeId, geneId, ensObjectId } = props;
+  const [isCollapsed, setIsCollapsed] = useState(true);
+  const { data } = useGetTrackPanelGeneQuery({
+    genomeId,
+    geneId
+  });
+  const trackStatus = useSelector((state: RootState) =>
+    getBrowserTrackState(state, {
+      genomeId,
+      objectId: ensObjectId,
+      tracksGroup: 'objectTracks',
+      categoryName: 'main',
+      trackId: GENE_TRACK_ID
+    })
+  );
+  const dispatch = useDispatch();
+
+  if (!data) {
+    return null;
+  }
+
+  const toggleExpand = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  const onShowMore = () => {
+    dispatch(
+      changeDrawerViewForGenome({
+        genomeId,
+        drawerView: {
+          name: 'gene_summary',
+          payload: {
+            geneId: geneId
+          }
+        }
+      })
+    );
+    dispatch(toggleDrawerForGenome({ genomeId, isOpen: true })); // FIXME - can this be deleted?
+  };
+
+  const toggleTrack = ({
+    trackId,
+    status
+  }: {
+    trackId: string;
+    status: Status;
+  }) => {
+    const newStatus =
+      status === Status.SELECTED ? Status.UNSELECTED : Status.SELECTED;
+    updateGenomeBrowser({ trackId, status: newStatus });
+
+    dispatch(
+      updateTrackStatesAndSave({
+        [genomeId]: {
+          objectTracks: {
+            [ensObjectId]: {
+              main: {
+                [trackId]: newStatus
+              }
+            }
+          }
+        }
+      })
+    );
+  };
+
+  const updateGenomeBrowser = ({
+    trackId,
+    status
+  }: {
+    trackId: string;
+    status: Status;
+  }) => {
+    const currentTrackStatus = status === Status.SELECTED ? 'on' : 'off';
+
+    const payload = {
+      [currentTrackStatus]: trackId
+    };
+
+    browserMessagingService.send('bpane', payload);
+  };
+
+  const { gene } = data;
+  const sortedTranscripts =
+    isEnvironment([Environment.PRODUCTION]) || isCollapsed
+      ? [defaultSort(gene.transcripts)[0]]
+      : defaultSort(gene.transcripts);
+
+  return (
+    <>
+      <GroupTrackPanelItemLayout
+        isCollapsed={isCollapsed}
+        visibilityStatus={trackStatus}
+        onChangeVisibility={() =>
+          toggleTrack({ trackId: GENE_TRACK_ID, status: trackStatus })
+        }
+        onShowMore={onShowMore}
+        toggleExpand={toggleExpand}
+      >
+        <div className={styles.label}>
+          <span className={styles.labelTextStrong}>
```

Comment:
`labelTextStrong` -> `labelStrong`?

Reply:
In the styles, there is also `.labelTextSecondary` and `.labelTextSecondaryStrong`. Without "text", `labelSecondary` doesn't sound quite right to me.